server/models/index.js

```diff
@@ -0,0 +1,35 @@
+import dotenv from 'dotenv';
+import Client from '../config/index';
+
+import { tables,recordUser } from './tableQueries';
+
+dotenv.config();
+Client.connect();
+// Run table queries
+export const createTables = () => {
+  
+  Client.query(tables)
+    .then(() =>{
+      Client.end();
+    })
+    .catch();
+};
+export const insertadmin = () => {
+  
```

Comment:
This method is similar to the one above `createTables`. Make it reusable by using parameters.